cmd/test/report.yaml

```diff
@@ -82,4 +82,7 @@ results:
     type: Mandatory
     outcome: PASS
     reason: Chart does not contain CRDs
-
+  - check: v1.0/no-missing-required-annotations
```

Comment:
can we avoid the double negative, instead of "no missing required annotations" I would say "required annotations present"